components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Storage/PublishedNodesJobConverterTests.cs

```diff
@@ -406,7 +412,7 @@ private class StandaloneIdentity : IIdentity {
 
             Assert.NotEmpty(jobs);
             Assert.Single(jobs);
-            Assert.Equal("i=2258", jobs
+            Assert.Equal(null, jobs
```

Comment:
Same comment as above, don't we expect that `i=2258` will be present in `jobs` ? Can we please check that it is still there ?

Reply:
I changed the logic a bit so that Id stores DataSetFieldId or DisplayName if any is configured. The NodeId itself is stored into PublishedVariableNodeId. That's why here the test is to validate that the Id is null.